frontend/public/components/modals/labels-modal.jsx

```diff
@@ -107,8 +107,9 @@ export const podSelectorModal = createModalLauncher((props) => (
     isPodSelector={true}
     // t('public~Pod selector')
     descriptionKey="public~Pod selector"
-    // t('public~Determines the set of pods targeted by this {{kind: props.kind.label.toLowerCase()}}.')
-    messageKey="public~Determines the set of pods targeted by this {{kind: props.kind.label.toLowerCase()}}."
+    // t('public~Determines the set of pods targeted by this {{kind}}.')
+    messageKey="public~Determines the set of pods targeted by this {{kind}}."
+    messageVariables={{ kind: props.kind.label.toLowerCase() }}
```

Comment:
We should check if there is a `kind.labelKey` and provide that if available over the `kind.label`.

Reply:
Hi @rebeccaalpert, do you mean like this:
```
   messageKey={'public~Determines the set of pods targeted by this {{kind}}.'}
   messageVariables={{ kind: props.kind.labelKey || props.kind.label.toLowerCase() }}
 ```
Doesn't seem right considering what I'm doing [here](https://github.com/openshift/console/pull/9701/files#diff-c80ff630ac928e3ae15ee177fdfc144b18986a640ef7062763feff116b0197b8R62)? -thanks

Reply:
Yeah, we'd have to do something like `messageVariables={{ kind: props.kind.labelKey ? t(props.kind.labelKey) : props.kind.label.toLowerCase() }}`.

My concern is that we're not passing the lower-cased kind to translation at all. For example, in the Chinese language files, a service is localized as "服务." It seems like it would be confusing to have it as "服务" in some contexts and "service" in others.

Reply:
Ok, updated, thanks, now looks like: 
![image](https://user-images.githubusercontent.com/12733153/128724657-51e9d505-8a5c-442d-ae01-53ae70e82bf9.png)